errorprone-checks/src/main/java/tech/pegasys/errorpronechecks/BannedMethod.java

```diff
@@ -34,12 +34,15 @@
 @BugPattern(
     name = "BannedMethod",
     summary = "Some methods should not be used, make sure that doesn't happen.",
-    severity = WARNING)
+    severity = WARNING,
+    linkType = LinkType.NONE)
 public class BannedMethod extends BugChecker implements MethodInvocationTreeMatcher {
 
   private static final ImmutableMap<Matcher<ExpressionTree>, String> BANNED_METHOD_LIST =
       ImmutableMap.of(
-          allOf(staticMethod().onClass("com.google.common.base.Objects").withAnyName()),
+          staticMethod().onClass("java.lang.System").named("currentTimeMillis"),
```

Comment:
Drop the ban hammer here.

![Ban Hammer](https://gatherer.wizards.com/Handlers/Image.ashx?multiverseid=466977&type=card)